src/Eccube/Repository/CalendarRepository.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+/*
+ * This file is part of EC-CUBE
+ *
+ * Copyright(c) EC-CUBE CO.,LTD. All Rights Reserved.
+ *
+ * http://www.ec-cube.co.jp/
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Eccube\Repository;
+
+use Carbon\Carbon;
+use Doctrine\ORM\NoResultException;
+use Doctrine\ORM\OptimisticLockException;
+use Doctrine\ORM\ORMException;
+use Eccube\Entity\Calendar;
+use Symfony\Bridge\Doctrine\RegistryInterface;
+
+/**
+ * CalendarRepository
+ *
+ * This class was generated by the Doctrine ORM. Add your own custom
+ * repository methods below.
+ */
+class CalendarRepository extends AbstractRepository
+{
+    /**
+     * CalendarRepository constructor.
+     */
+    public function __construct(RegistryInterface $registry)
+    {
+        parent::__construct($registry, Calendar::class);
+    }
+
+    /**
+     * @param int $id
+     *
+     * @return Calendar
+     * @throws \Exception
+     */
+    public function get($id = 1)
+    {
+        $calendar = $this->find($id);
+
+        if (null === $calendar) {
+            throw new \Exception('Calendar not found. id = '.$id);
+        }
+
+        return $this->find($id);
```

Comment:
```suggestion
        return $calendar;
```

`return $calendar;` で良いと思います